LICENSE

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
I believe this is to be Apache 2.0, not AGPL, since it's a client shim and the clients are Apache 2.0. @amastracci?

Reply:
Confirmed. All client libs should be licensed under apache2.0

Reply:
@amastracci Then the source code will also need the licence cleaned up. @paularchard please create a task for this.